core/src/main/java/com/xmartlabs/bigbang/core/helper/GeneralErrorHelper.java

```diff
@@ -31,69 +30,65 @@
       EntityNotFoundException.class,
       UnknownHostException.class
   );
-  private final StackTraceElement DUMMY_STACK_TRACE_ELEMENT = new StackTraceElement("", "", null, -1);
-  private final StackTraceElement[] DUMMY_STACK_TRACE_ELEMENT_ARRAY =
-      new StackTraceElement[] {DUMMY_STACK_TRACE_ELEMENT};
-
+  private final StackTraceElement DUMMY_STACK_TRACE_ELEMENT =
+      new StackTraceElement("General error logger", "", null, -1);
   private final Map<Class<? extends Throwable>, Consumer<? super Throwable>> throwableHandlers = new HashMap<>();
 
-  private final BuildInfo buildInfo;
-
   @Inject
```

Comment:
you need inject the class, If I remove it I have to make a provider. makes sense?